tools/transformer/src/atlasTransformations.js

```diff
@@ -107,15 +107,22 @@ module.exports = function runTransformations(openapi) {
 };
 
 function workaroundNestedTransformations(openapi) {
+  var parentObject;
   try {
-    const parentObject = getObjectFromYamlPath(
+    parentObject = getObjectFromYamlPath(
       ".components.schemas.RegionConfig",
       openapi
     );
-    if (parentObject) {
-      transformOneOfProperties(parentObject, openapi);
-    }
-  } catch (e) {
+  } catch (e) {}
+  try {
+    parentObject = getObjectFromYamlPath(
```

Comment:
```suggestion
    parentObject = parentObject || getObjectFromYamlPath(
```

if parentObject is successfully set at line 112, then there's no need to set it here again, right?